packages/base/Menu/src/Menu/Menu.tsx

```diff
@@ -62,13 +109,13 @@ export const Menu = forwardRef<HTMLUListElement, Props>(function Menu(
               onClick={onBackClick}
             >
               <Typography size='xsmall' color='dark-grey' variant='body'>
-                <BackMinor16 className={classes.backButtonIcon} />
+                <BackMinor16 className='mt-[-1px] mr-1 ml-[-5px] align-middle' />
```

Comment:
NIT (subjective proposal)


```suggestion
                <BackMinor16 className='-mt-[1px] mr-1 -ml-[5px] align-middle' />
```

Reply:
It looked weird to me when using arbitrary value and the `-` prefix 🤔

Reply:
but I don't have strong opinion against it, I will update it. ✅